xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -813,10 +813,10 @@ void CMusicInfoTag::Serialize(CVariant& value) const
 
   value["displayartist"] = GetArtistString();
   value["displayalbumartist"] = GetAlbumArtistString();
-  value["artistsort"] = GetArtistSort();
+  value["sortartist"] = GetArtistSort();
   value["album"] = m_strAlbum;
   value["albumartist"] = m_albumArtist;
-  value["albumartistsort"] = m_strAlbumArtistSort;
+  value["sortalbumartist"] = m_strAlbumArtistSort;
```

Comment:
These two changes look backwards incompatible. Or have these properties never been exposed to JSON-RPC?

Reply:
Although I tried to anticipate JSON use in the original PR that added this field, it has never been exposed. Hance switching to more consistent naming

Reply:
Personally I'd prefer if the database layout looks the same for a newly created database and an updated database even if there is no positional indexing right now.